bcp-003-03-certificate-provisioning.md

```diff
@@ -286,26 +287,21 @@ On start up or on change of network state the EST Client MUST attempt to discove
-- Use of 802.1AR - secure device identity for a distinct PKI for client Auth
-- Define EST Client support for the CSR endpoint on the EST Server
-- Define the use of Root CA Key Update, for Root CA rollover/renewal
-- Possible extensions to EST RFC
-  - Add method to signal certificate profile, eg. TLS Client Auth or TLS Server Auth
-  - Add method to signal profile for server side key generation, digital signature algorithm
-  - discover of EST API
+## Notes to Implementors
+
+- The certificate returned by the EST Server MAY not be valid until sometime in the future
+  - The 'Not Before' date MUST be checked before using the certificate
+- Care SHOULD be taken when implementing the certificate renewal process, to make sure the renewal does not affect user operation in a critical way (eg. loss of control by connected NMOS clients).
 
 ## Security Considerations
 
-Bootstrap Distribution of CA Certificate
-- It has been decided that EST Clients will implicitly trust EST Server's found using DNS-SD, no authentication of its TLS Certificate automatic or manual, needs to be performed.
-- However EST Clients MUST use the CA returned by the EST Server endpoint `/cacerts` to establish an Explicit Trust Anchor database used to for subsequent TLS authentication of the EST Server.
+Bootstrap Distribution of CA Certificate:
+- It has been decided that EST Clients MAY implicitly trust the EST Server found using DNS-SD, so no authentication of its TLS Certificate automatic or manual MUST to be performed when using the `/cacerts` endpoint.
+- EST Clients MUST use the CA returned by the EST Server endpoint `/cacerts` to establish an Explicit Trust Anchor database used for subsequent TLS authentication of the EST Server and NMOS devices.
 - This deviation from the EST specification has been allowed due to the minimal risk around trusting a rogue EST Server
-  - If an EST Client is provisioned an incorrect CA Certificate by the rogue EST Server the EST Client would trust NMOS Node or NMOS registries. It would not divulge sensitive information, but quite clearly not work.
-  - If an EST Client is provisioned with a TLS Certificate by the rogue EST Server, the EST Client would not be trusted by other NMOS Nodes and Controller on the network, and therefore not work correctly.
-  - This dispensation should not be applied to NMOS Controllers, as if they have the incorrect CA installed they would trust rogue devices.
+  - If an EST Client is provisioned with an incorrect CA Certificate by the rogue EST Server the EST Client would trust not NMOS Nodes or NMOS Registries and therefore not divulge sensitive information.
+  - If an EST Client is provisioned with a TLS Certificate by the rogue EST Server, the EST Client would not be trusted by other NMOS Nodes and Controllers on the network.
+  - This dispensation MUST not be applied to NMOS Controllers, as if they have the incorrect CA installed they would trust rogue devices.
```

Comment:
As this is another normative element it would be good to separate it and move it higher. Ie. NMOS Controllers MUST NOT implicitly trust the EST Server's TLS Cert... Ideally it would be good to avoid calling out Nodes/Registries/Controllers specifically as we may miss another class of device which doesn't exist yet. Preferably we would describe what characteristics controllers have which mean we don't want to allow this implicit trust.